lib/custom_lib/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2021, Legrand North America, LLC.
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library()
+zephyr_library_sources(custom_lib.c)
+zephyr_include_directories(${ZEPHYR_CURRENT_MODULE_DIR}/include/custom_lib)
```

Comment:
should not be needed now, since we have `zephyr_include_directories(include)`

Reply:
Accepted, and manually rolled in.